build.gradle

```diff
@@ -4,10 +4,10 @@ buildscript {
   ext.versions = [
     minSdk: 23,
     compileSdk: 33,
-    kotlin: '1.7.20',
-    composeUi: '1.3.1',
-    composeCompiler: '1.4.0-alpha01',
-    composeUiMaterial3: '1.0.1',
+    kotlin: '1.8.21',
+    composeUi: '1.5.0-alpha04',
+    composeCompiler: '1.4.7',
+    composeUiMaterial3: '1.2.0-alpha01',
```

Comment:
Any reason for depending on an alpha version of material3? Can we depend on `1.1.0` instead?

Reply:
Yep. 1.1.0 works as well.